# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1744064700 -7200
# Parent  e8c8f7d45ad174d124d7264cca230f7a5f4844f9
Branding: Adjust in app notifications (IAN) for Betterbird use.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -1442,9 +1442,9 @@ pref("dom.disable_window_flip", true);
 // Thunderbird.
 pref("browser.theme.dark-private-windows", true);
 
 // In-app notifications are disabled while the feature is being implemented.
-pref("mail.inappnotifications.enabled", true);
+pref("mail.inappnotifications.enabled", false);
 // Refresh interval for in-app notifications in ms (6 hours)
 pref("mail.inappnotifications.refreshInterval", 21600000);
 // In-app notification server endpoint is specified by branding prefs.
 
diff --git a/mail/branding/betterbird/content/inAppNotificationData.json b/mail/branding/betterbird/content/inAppNotificationData.json
new file mode 100644
--- /dev/null
+++ b/mail/branding/betterbird/content/inAppNotificationData.json
@@ -0,0 +1,16 @@
+[
+  {
+    "id": "End-of-year-2025",
+    "start_at": "2025-12-01T00:00:00Z",
+    "end_at": "2025-12-31T00:00:00Z",
+    "title": "End-of-year Appeal 2025",
+    "description": "",
+    "URL": "https://www.betterbird.eu/end-of-year/",
+    "CTA": "",
+    "severity": 3,
+    "type": "donation_tab",
+    "targeting": {
+      "percent_chance": 100
+    }
+  }
+]
diff --git a/mail/branding/include/release-prefs.js b/mail/branding/include/release-prefs.js
--- a/mail/branding/include/release-prefs.js
+++ b/mail/branding/include/release-prefs.js
@@ -33,5 +33,5 @@ pref("app.vendorURL", "https://www.bette
 
 pref("browser.search.param.ms-pc", "MOZT");
 
 // In-app notification server endpoint
-pref("mail.inappnotifications.url", "https://notifications.thunderbird.net/%IAN_SCHEMA_VERSION%/notifications.json");
+pref("mail.inappnotifications.url", "https://www.betterbird.eu/start/notifications/2.0/notifications.json");
